samples/MiniZinc/data.dzn

```diff
@@ -0,0 +1 @@
+n=3;
```

Comment:
Ah, darnit. Ok, I can see two options:

- I could inline the license into the source code of the sample files. But I presume this would confuse the classifier.
- I could add a `vendor/licenses/samples` folder, add the license there, and then reference it from a short inline comment in the files.